arbnode/seq_coordinator.go

```diff
@@ -663,15 +663,6 @@ func (c *SeqCoordinator) update(ctx context.Context) (time.Duration, error) {
 		}
 		c.prevRedisMessageCount = prevRemoteMsgCount
 	}
-	// Cache the previous redis coordinator's message count
-	if c.prevRedisCoordinator != nil && c.prevRedisMessageCount == 0 {
```

Comment:
nit: why are we deleting this code?

Reply:
@ImJeremyHe  same for this, its present here: https://github.com/celestiaorg/nitro/blob/v3.6.7/arbnode/seq_coordinator.go#L658